From c39d7c9c0a39efe5920bd8e384820e4813b54de7 Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Tue, 14 Aug 2012 18:33:45 +0800
Subject: ENGR00219872-1 MX6Q Sabresd iomux:Add LVDS CABC_EN0/1 support

This patch configures NANDF_CS2/3 to be GPIO_6_15/16 to
support LVDS CABC_EN0/1.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
(cherry picked from commit fdff66991738a56a7e1bc735cf452d57f1771c13)
---
 arch/arm/mach-mx6/board-mx6q_sabresd.h |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabresd.h b/arch/arm/mach-mx6/board-mx6q_sabresd.h
index 54ad3ac..da26cd9 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabresd.h
+++ b/arch/arm/mach-mx6/board-mx6q_sabresd.h
@@ -259,6 +259,10 @@ static iomux_v3_cfg_t mx6q_sabresd_pads[] = {
 	MX6Q_PAD_NANDF_CS0__GPIO_6_11,
 	/* DISP_PWR_EN */
 	MX6Q_PAD_NANDF_CS1__GPIO_6_14,
+	/* CABC_EN0 */
+	MX6Q_PAD_NANDF_CS2__GPIO_6_15,
+	/* CABC_EN1 */
+	MX6Q_PAD_NANDF_CS3__GPIO_6_16,
 };
 
 static iomux_v3_cfg_t mx6q_sabresd_csi0_sensor_pads[] = {
-- 
1.7.9.5

